oidc_provider/lib/endpoints/introspection.py

```diff
@@ -85,6 +85,7 @@ def create_response_dic(self):
                 response_dic[k] = self.id_token[k]
         response_dic['active'] = True
         response_dic['client_id'] = self.token.client.client_id
+        response_dic['scope'] = self.client.scope
```

Comment:
why it's client's scope? Should be the token scope. 

 Assume we have 2 clients:
* client_1 with scopes (read write)
* client_2 with scope (token_introspection)

client_1 requests token, and passed it to a different app. Then client_2 validates the token passed with introspection and expects scope `read write` not `token_introspection`. 

This scenario covered in https://github.com/juanifioren/django-oidc-provider/pull/329.